src/Microsoft.Health.Dicom.Blob/Features/Storage/BlobWorkitemStore.cs

```diff
@@ -92,6 +94,33 @@ await using (Stream stream = _recyclableMemoryStreamManager.GetStream(AddWorkite
             }
         }
 
+        /// <inheritdoc />
+        public async Task<DicomDataset> GetWorkitemAsync(WorkitemInstanceIdentifier workitemInstanceIdentifier, CancellationToken cancellationToken)
+        {
+            EnsureArg.IsNotNull(workitemInstanceIdentifier, nameof(workitemInstanceIdentifier));
+            BlockBlobClient cloudBlockBlob = GetBlockBlobClient(workitemInstanceIdentifier);
```

Comment:
Are we using a different container for workItem blobs?

Reply:
Yes we are using different container for workitem